Makefile

```diff
@@ -121,17 +129,39 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 	rm Dockerfile.cross
 
 .PHONY: build-installer
-build-installer: manifests generate kustomize ## Generate a consolidated YAML with CRDs and deployment.
+build-installer: update-prefix manifests generate kustomize ## Generate a consolidated YAML with CRDs and deployment.
 	mkdir -p dist
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default > dist/install.yaml
+	@echo -e "---" >> dist/install.yaml
+	$(KUSTOMIZE) build config/csi-rbac >> dist/install.yaml
 
 ##@ Deployment
 
 ifndef ignore-not-found
   ignore-not-found = false
 endif
 
+PHONY: update-prefix
+update-prefix: backup-prefix ## Update the kustomization.yaml with the new prefix
+	@echo "Updating prefixes in kustomization.yaml files..."
+	sed -i.bak "s/namePrefix: .*/namePrefix: ${KUSTOMIZE_NAME_PREFIX}/" config/default/kustomization.yaml
+	sed -i.bak "s/namespace: .*/namespace: ${NAMESPACE_PREFIX}/" config/default/kustomization.yaml
+	sed -i.bak "s/namePrefix: .*/namePrefix: ${KUSTOMIZE_CSI_NAME_PREFIX}/" config/csi-rbac/kustomization.yaml
+	sed -i.bak "s/namespace: .*/namespace: ${CSI_NAMESPACE_PREFIX}/" config/csi-rbac/kustomization.yaml
```

Comment:
Why use a sed? The entire point of using kustomize is to have no need for text replacement operations.
If you need to apply additional kaustomize options based on you own values, you need to write a new file with the required changes then apply it as the root of your kustomize operation

Reply:
This is to have flexibility where user need not to worry about customization and just make use of makefile, am fine to drop it as well if not required.

Reply:
Thanks @nb-ohad for suggesting `kustomize edit set nameprefix` i will remove sed